src/controllers/space-controller.js

```diff
@@ -15,6 +15,10 @@ function SpaceController(el, parent) {
   //   // is a brand new Space component.
   //   window.location.reload();
   // }
+  var self = this,
+    reinitialize = _.debounce(function () {
```

Comment:
Looks like there's a mix of fat arrow functions and storing the context of `this`. I would stick with one implementation.